homeassistant/components/recorder/__init__.py

```diff
@@ -128,7 +128,6 @@ def is_entity_recorded(hass: HomeAssistant, entity_id: str) -> bool:
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the recorder."""
-    hass.data[DOMAIN] = {}
```

Comment:
Removed because this is now done by calling `helpers.recorder.async_initialize_recorder`